data/soapvalidator/MailClient/Search/Bugs/bug72260.xml

```diff
@@ -85,7 +85,7 @@
 		<t:test>
 			<t:request>
 				<SearchRequest xmlns="urn:zimbraMail" types="message">
-					<query>"to:kevinh"</query>
+					<query>to:"kevinh"</query>
```

Comment:
Does this work on legacy and zimbraX both?

Reply:
yes , this is working fine in zimbrax and Legacy